compiler/ml/ast_mapper_to0.ml

```diff
@@ -407,6 +407,18 @@ module E = struct
     | Pexp_open (ovf, lid, e) ->
       open_ ~loc ~attrs ovf (map_loc sub lid) (sub.expr sub e)
     | Pexp_extension x -> extension ~loc ~attrs (sub.extension sub x)
+    | Pexp_jsx_fragment (o, xs, c) ->
```

Comment:
There should be a corresponding change in `ast_mapper_from0` that inverts the conversion.
Also, some extension of the test cases in `tests/tools_tests/ppx/TestPpx.res` to check that back and forth conversion works fine.

Reply:
Alright, I think the new tests from https://github.com/rescript-lang/rescript/pull/7318 prove this now.